app/views/shared/_commodity_tree.html.erb

```diff
@@ -14,7 +14,7 @@
       </div>
     </div>
     <ul class="commodities js-commodities">
-      <%= render commodities %>
+      <%= render commodities, initial_indent: commodities.first.number_indents %>
```

Comment:
What happens when there are no commodities? This will trigger an exception presumably - can this be `commodities.first&.number_indents` instead - thanks

Reply:
I've just checked, and this is something that does not happen now, thanks to the `rescue_from Faraday::ResourceNotFound` on `GoodsNomenclaturesController`.

Anyway, it's a change that makes sense, I am happy to do it, and default `initial_indent` to one, in case of empty commodities.